CONTRIBUTING.md

```diff
@@ -27,6 +27,23 @@ the title and let us know when it’s ready to review in the comments.
 Valora is part of the Celo community and follows Celo's [Code of
 Conduct](https://celo.org/code-of-conduct).
 
+### Contributing _substantial_ changes
+
+A substantial change is one that significantly alters the user
+experience or an underlying implementation. For example, we would
+consider the following as examples of substantial changes:
+
+- Adding support for in-app swaps.
+- Changing how Valora calculates asset value in local fiat.
+- Changing Valora's branding (_e.g.), color palette).
+- Removing screens from on-boarding.
```

Comment:
these don't seem likely for 3rd parties to pursue

Reply:
maybe "adding a way for Valora users to go through decentralized KYC", since we've heard people interested in something like that